### PR TITLE
EOS-21921: addressed integration issues

### PR DESCRIPTION
**Pull request by tanujashinde0405**
Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

### CODE REVIEW

**Review by nitin-seagate — APPROVED** [review]
Looks good to me...

**Review by mazinamdar — APPROVED** [review]

File: lr-cli/cortx_setup/commands/cluster/create.py [hunk 1]
```diff
@@ -195,9 +195,6 @@ def run(self, **kwargs):
             )
             PillarSync().run()
 
-            self.logger.debug("Refreshing machine id on the system")
```
Comment (divya-vijay) [inline lr-cli/cortx_setup/commands/cluster/create.py:198]: Please update empty cluster validation in `cluster_id.py` along with the same PR.

Comment (tanujashinde0405) [reply lr-cli/cortx_setup/commands/cluster/create.py:198]: done

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Review by divya-vijay — APPROVED** [review]

**Pull request merged**